assignments/callbacks.js

```diff
@@ -21,28 +21,78 @@ const items = ['Pencil', 'Notebook', 'yo-yo', 'Gum'];
 
 */
 
+const items = ['Pencil', 'Notebook', 'yo-yo', 'Gum'];
 
 function getLength(arr, cb) {
-  // getLength passes the length of the array into the callback.
+  return cb(arr);
 }
 
-function last(arr, cb) {
-  // last passes the last item of the array into the callback.
+function lengthOfArray(arr){
+  console.log(arr.length);
 }
 
+getLength(items, lengthOfArray);
+
+  // getLength passes the length of the array into the callback.
+
+  function last(arr, cb) {
+    return cb(arr);
+  }
+  
+  function lastOfArray(arr){
+    console.log(arr.slice(-1)[0]);
+  }
+  
+  last(items, lastOfArray);
+
+  // last passes the last item of the array into the callback.
+
+
 function sumNums(x, y, cb) {
+  return cb(x, y);
   // sumNums adds two numbers (x, y) and passes the result to the callback.
 }
 
+function add(x, y){
+  console.log(x + y);
+}
+
+sumNums(2,2,add);
+
 function multiplyNums(x, y, cb) {
+  return cb(x, y);
   // multiplyNums multiplies two numbers and passes the result to the callback.
 }
 
+function multiply(x, y){
+  console.log(x * y);
+}
+
+multiplyNums(3, 4, multiply);
+
+
 function contains(item, list, cb) {
+  return cb(item, list);
+  
   // contains checks if an item is present inside of the given array/list.
   // Pass true to the callback if it is, otherwise pass false.
 }
 
+function match(item, list) {
+  for (var i = 0; i < list.length; i++) {
+    if (list[i] === item) {
+      console.log("true");
```

Comment:
for `functions`, you might want to use `return` to stop the `for loop`.
Example:
`return true;`